src/Paket/Program.fs

```diff
@@ -745,9 +749,14 @@ let main() =
     use consoleTrace = Logging.event.Publish |> Observable.subscribe Logging.traceToConsole
     let paketVersion = AssemblyVersionInformation.AssemblyInformationalVersion
 
-
-
     try
+    let args = Environment.GetCommandLineArgs()
+    if args.Length = 1 && args.[0] = "restore" || args.Length = 2 && args.[0] = "--from-bootstrapper" && args.[1] = "restore" then
```

Comment:
```F#
match args with
| [| "restore" |]
| [| "--from-bootstrapper" ; "restore" |]
```

?

Reply:
yes looks better but might be slower :P 
The main problem is generating the sha512 hash in msbuild anyway ;) I don't think what I want is even possible...

Reply:
Ok fixed :)